string species
sensor_msgs/Image image
string location
string ring_color
string detection_time